### DISCUSSION TITLE
Units stiffness in hinge joint

### DISCUSSION THREAD

**Discussion by tomvdm1999** [google-deepmind/mujoco#157] [Asking for Help] [answered]
I was wondering what the default units were when defining stiffness in a hinge joint. I know MuJoCo is unitless and it is up to the programmer to keep the units consistent. The rest of my file all follows the SI units convention but I just don't know what SI units are used to define a hinge joint's stiffness.

**Comment by yuvaltassa** [comment, marked as answer]
They would be Newton-Metre/radian.
